insights-ansible-playbook-verifier.spec

```diff
@@ -0,0 +1,42 @@
+Name:     insights-ansible-playbook-verifier
+Version:  0.1.0
+Release:  0%{dist}
+Summary:  Ansible playbook verifier for Red Hat Insights
+License:  MIT
+
+URL:      https://github.com/RedHatInsights/%{name}
+# Source0:  %{url}/archive/refs/tags/v%{version}.tar.gz
+Source0:  %{name}-%{version}.tar.gz
+
+BuildArch: noarch
+Requires:  python3
+Requires:  python3-pyyaml
+
+BuildRequires: python3-devel
+%generate_buildrequires
+%pyproject_buildrequires
+
+%description
+insights-ansible-playbook-verifier package cryptographically verifies Ansible
+playbooks generated by Red Hat Insights.
+
+%prep
+%autosetup -p1
+
+%build
+%pyproject_wheel
+
+%install
+%pyproject_install insights_ansible_playbook_lib insights_ansible_playbook_verifier
+%pyproject_save_files insights_ansible_playbook_lib insights_ansible_playbook_verifier
+
+%check
+%pyproject_check_import insights_ansible_playbook_lib insights_ansible_playbook_verifier
+
+%files -n %{name} -f %{pyproject_files}
+# TODO Save the executable into /usr/libexec/ instead
+%{_bindir}/%{name}
```

Comment:
this can be easily solved by:
a) adding this between `%pyproject_install` and `%pyproject_save_files`:
```spec
mkdir -p %{buildroot}%{_libexecdir}
mv %{buildroot}%{_bindir}/%{name} %{buildroot}%{_libexecdir}
```
b) changing this to use libexec:
```spec
%{_libexecdir}/%{name}
```

of course, the ideal way should be tweaking setuptools to use libexec as install location (which would avoid (a)); in the meanwhile, the above changes should do it